disperser/apiserver/server_v2.go

```diff
@@ -232,6 +257,9 @@ func (s *DispersalServerV2) RefreshOnchainState(ctx context.Context) error {
 }
 
 func (s *DispersalServerV2) GetPaymentState(ctx context.Context, req *pb.GetPaymentStateRequest) (*pb.GetPaymentStateReply, error) {
+	if s.meterer == nil {
```

Comment:
why do we check nil meterer here instead of in the `NewDispersalServerV2` constructor?

Reply:
This is to get around with disabling payments. Currently, payment's disabled and meterer is not set. We still want the server to run until we reenable it